agent/test-agent-cli/examples/example_test_agent_cli/example-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Get the configuration details for test and save it in a JSON file.
+# Set the Task_state for test as running(This can be checked in Kubectl describe test or testsys status).
+test-agent-cli init > ExampleConfig.json
+
+echo "starting the Example test"
+
+person=$(jq '.person' ExampleConfig.json)
+helloCount=$(jq '.helloCount' ExampleConfig.json)
+helloDurationMilliseconds=$(jq '.helloDurationMilliseconds' ExampleConfig.json)
+
+echo "$person"
+echo "$helloDurationMilliseconds"
+echo "$helloCount"
```

Comment:
nit: to be more consistent
```suggestion
echo "${person}"
echo "${helloDurationMilliseconds}"
echo "${helloCount}"
```